FhirLoader.Tests.E2E/UnitTest.cs

```diff
@@ -0,0 +1,83 @@
+using FhirLoader.Tool;
+
+namespace FhirLoader.Tests.E2E
+{
+    public class UnitTest
```

Comment:
Since these tests are CLI specific, E2E tests, can we change the folder structure, project name, and class to match?

-`FHIRLoader.Tool.Tests.E2E` would be a good project/folder structure.
- Class/filename should be E2E not unit. Feel free to split these up if it makes sense.

Reply:
Renamed solution to FHIRLoader.Tool.Tests.E2E and file to FHIRLoaderE2ETest.